Yuyan-Lei/NewsFlex-chrome-extension
# use GPT api for summaries, simplifier and translator

Just summary is supported for now.

## Testing
- Navigate to chrome://extensions
- Click load unpacked and use the root directory of this project
- Click the extension and inspect the popup
- Toggle the bottom checkbox and the output should console log

## Notes
We probably want to only want to call the API when the checkbox goes from unchecked to checked
